scipy/stats/tests/test_stats.py

```diff
@@ -5350,6 +5350,243 @@ def test_std_zero(self):
         assert_equal(stats.sigmaclip(x)[0], x)
 
 
+class TestAlexanderGovern:
+    def test_compare_dtypes(self):
+        args = [[13, 13, 13, 13, 13, 13, 13, 12, 12],
+                [14, 13, 12, 12, 12, 12, 12, 11, 11],
+                [14, 14, 13, 13, 13, 13, 13, 12, 12],
+                [15, 14, 13, 13, 13, 12, 12, 12, 11]]
+        args_int16 = np.asarray(args, dtype=np.int16)
+        args_int32 = np.asarray(args, dtype=np.int32)
+        args_uint8 = np.asarray(args, dtype=np.uint8)
+        args_float64 = np.asarray(args, dtype=np.float64)
```

Comment:
No need for `asarray`...
```suggestion
        args_int16 = np.array(args, dtype=np.int16)
        args_int32 = np.array(args, dtype=np.int32)
        args_uint8 = np.array(args, dtype=np.uint8)
        args_float64 = np.array(args, dtype=np.float64)
```